### ISSUE TITLE
Kodi 18.6 "Mark as watched" checkmark not showing

### ISSUE THREAD

**Issue by islandtj** [Wilson-Magic/plugin.video.hummingbird#23]
Aloha @Wilson-Magic 
Big mahalo to your efforts in this awesome addon.  I am a huge fan of your Masterani Redux addon, nice to see you have created another great anime addon.  As the header indicates, shows that have been watched in kodi are not showing the "watched" checkmark after complete viewing.  Even when I attempt to manually force the checkmark to show in the menu/options section, it still comes up blank.  Note, I do see the "play" symbol, if I leave a show partway and it still shows when I return.  Is there a settings options I need to activate to have the watched feature show?  Thanks again!  I am on version 0.5.0 of Hummingbird.

**Comment by Goldenfreddy0703** [comment]
Hey man, sorry for bothering you but its been like 18 days sense the dev responded and I dont know what happened to him tbh. By anychance, does the addon work for you and is their any playback errors?

**Comment by islandtj** [comment]
> Hey man, sorry for bothering you but its been like 18 days sense the dev responded and I dont know what happened to him tbh. By anychance, does the addon work for you and is their any playback errors?

Yes, it works but I only get successful playbacks using AnimeFlix and AnimeTwist hosters.  I mainly select and use AnimeTwist. In the addon it shows a select of 480 resolution for AnimeTwist  but I actually get 1080 playback.

**Comment by Goldenfreddy0703** [comment]
Ok I'm glad, I'm getting playback errors for some reason and idk why even tho i configured everything in the settings.

**Comment by Wilson-Magic** [comment]
Hi, that checkmark only automatically displays in the my accounts section. Is that where you are looking? It should automatically show it when you watch something elsewhere anyway, that's just a feature of kodi.

**Comment by islandtj** [comment]
> Hi, that checkmark only automatically displays in the my accounts section. Is that where you are looking? It should automatically show it when you watch something elsewhere anyway, that's just a feature of kodi.

Mahalo, it's actually the Kodi "watched" feature that's not working in Hummingbird. So if I don't sign in to an account, the Kodi feature should still work but it doesn't.

**Comment by b1n4ryk1lla** [comment]
same here the results list arent showing the episodes watched (so you know where to pick back up) as part of kodi like say when you watch a series in another addon or in the past WS addon does
